packages/frontend/src/pages/Models.svelte

```diff
@@ -145,6 +147,15 @@ async function importModel() {
               {/if}
             </Route>
 
+            <!-- Imported models -->
+            <Route path="/imported">
+              {#if importedModels.length > 0}
+                <Table kind="model" data="{importedModels}" columns="{columns}" row="{row}"></Table>
+              {:else}
+                <div role="status">There is no model yet</div>
```

Comment:
@cdrage suggests to use There are no models yet

Reply:
This is a copy paste of the other tab, another PR would be required for the others tab as this is not the scope